### DISCUSSION TITLE
Pages deploys to my private domain but it's not associated with github anywhere

### DISCUSSION THREAD

**Discussion by sorinpetrov** [community/community#50922] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all,

I just started out using github more and I wanted to host my project on pages. For some reason it deploys to sorinpetrov.com although I cannot find anywhere where that domain is written down. sorinpetrov.github.io also for some reason redirects to sorinpetrov.com. I want to disassociate these 2 domains but I can't find how or where.

In the repository pages settings the custom domain is blank.

![image](https://user-images.githubusercontent.com/9077282/227527651-ee0eb28e-e44c-4631-b93e-2bc4b0ddbaa8.png)

![image](https://user-images.githubusercontent.com/9077282/227527812-ec2cf1a2-8d0a-46db-82cd-16cc07ba0020.png)

in the domain manager there's also no association with github anywhere in dns or redirects.

Any ideas?

**Comment by enviGit** [comment]
### Hi Sorin!
You have already checked Custom domain field. Here are a few additional things you can check:

1. Check if you have a CNAME file in your repository root. If you do, make sure it contains the correct custom domain you want to use, without any typos or additional characters.
2. Check the DNS settings for your custom domain. Make sure that the domain is pointing to the correct IP address for GitHub Pages. You can find the correct IP addresses in the GitHub documentation.
3. Check the domain registrar settings for your custom domain. Make sure that there are no conflicting settings or redirects that could cause the issue.
4. Clear your browser cache and try accessing your site again. It's possible that your browser has cached an old redirect.
